### ISSUE TITLE
SDL Joystick Support

### ISSUE THREAD

**Issue by qreeves** [red-eclipse/base#65]
While this has never been a high priority (this is a twitch shooter after all), there are often requests for joystick/gamepad support. This issue is here if ever anyone wants to try to implement that.

**Comment by acz13** [comment]
Isn't this already possible with glovepie?

**Comment by shirepirate** [comment]
i don't even... xD
and yeah it's already possible through 3rd party apps (like xpadder for windows).
though i guess a free and integrated solution couldn't hurt, to cover all bases.

**Comment by freezurbern** [comment]
Relevant: http://quadropolis.us/node/571 - "Game Controller Support", for Spring Edition of Sauerbraten. Might be useful for inspiration. btw lol @ sauermod reference in comments.

**Comment by qreeves** [comment]
Yeah, please don't use SauerJoy as a base, it is way out of date and was never written properly to begin with (as far as Cube code formatting guidelines goes anyway).

**Comment by acerspyro** [comment]
Take a look at this thread http://redeclipse.net/forum/viewtopic.php?f=4&t=54
We talked about controllers in there, and I proposed a mapping scheme as well. I don't know how well it stands today, and I feel like the younger me could be quite a lot dumber. :D

**Comment by WeebNetsu** [comment]
I would like to bump this issue, joystick support would be a great feature
